src/views/Item/ItemImage.js

```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react'
+import { META } from '../../lib'
+import Image from '../../elements/Image/Image'
+
+function ItemImage(props) {
+  return <Image {...props} />
```

Comment:
@levithomason I have problem there.

1. I think I need use `createImage` factory, but will it pass props?
2. `ItemImage` doesn't have `ui` class by default. How we can deal with this?

![_357](https://cloud.githubusercontent.com/assets/14183168/18027928/453f0cf6-6c79-11e6-92f8-427dbd6b6ae4.png)